src/docs/releases/2.1.0.md

```diff
@@ -4,6 +4,12 @@ Release date: Not yet released
 
 ## Change Logs
 
-### New 'Azure Communication SMS' feature
+### New **Azure Communication Services (SMS)** Feature
 
 A new feature was added to allow you to send SMS messages using Azure Communication Services (ACS). Simply enable it then navigate to the admin dashboard > `Configurations` >> `Settings` >> `SMS` to configure the provider. For more information you can refer to the [docs](../reference/modules/Sms.Azure/README.md).
+
+### **Azure Communication Services (Email)** Feature Update
+
+The feature formerly known as **Azure Email Provider** has been renamed to **Azure Communication Services (Email)**.
+
+Additionally, the configuration provider key for the default provider has changed from `OrchardCore_Email_Azure` to `OrchardCore_Email_AzureCommunicationServices`. While the old key (`OrchardCore_Email_Azure`) will continue to bind for backward compatibility, it is scheduled for removal in the next major release. To ensure future compatibility, it is highly recommended to update your configuration to use the new key.
```

Comment:
```suggestion
No immediate action is required. However, additionally, the configuration provider key for the default provider has changed from `OrchardCore_Email_Azure` to `OrchardCore_Email_AzureCommunicationServices`. While the old key (`OrchardCore_Email_Azure`) will continue to bind for backward compatibility, it is scheduled for removal in the next major release. To ensure future compatibility, it is highly recommended to update your configuration to use the new key.
```

Reply:
I feel that my previous statement is more clear.

Reply:
These release notes are long, and we need to convey the most important points in one-two sentences for each feature, because nobody will read the whole thing, and they'll only scan for things that affect them. The most important is: do I need to do anything to keep my site working?

Reply:
When you start the paragraph with "No immediate action is required.", that is as far are people will read so it's counter productive. In this case, there is no breaking change so if they chose not to read, then "oh well". There isn't much value in this change any way. When we release v3, we'll then remove the key `OrchardCore_Email_Azure` and document it in the breaking changes and hopefully people will read it then (if not, that's their problem).

We provide this change logs as information only and there is no action needed from the developer "until we introduce any breaking changes in v3. The idea behind the change logs is to have a summary info for changes if something want to stay up to date and if we need to recall all the changes in the form of summary.

Reply:
OK!